README.md

```diff
@@ -64,7 +66,9 @@ Convert the following create method in Mongoose to ActiveRecord.
 
 How does module.exports help us with separation of concerns?
 
-```text
+```module.exports is the thing (sometimes an object) that is returned as the result of a require call.
+
+
```

Comment:
But why does it help with separation of concerns? Because it allows us to encapsulate related functionality into a single unit of code.